index.html

```diff
@@ -72,194 +72,146 @@ <h2 class="connect" href="#">LET'S CONNECT</h2>
       </div>
     </section>
 
-    <section class="work-container" id="containers">
-      <div class="container1" id="container1" data-modal-target="#modal1">
-        <img
-          class="desktop-image"
-          src="Images/Desktop-img1.png"
-          alt="Project Image"
-        />
-        <img
-          class="snapshoot"
-          src="Images/Snapshoot1.png"
-          alt="Project Image"
-        />
-        <div class="daily">
-          <h3 class="Tonic">Tonic</h3>
-          <ul class="offer">
-            <li class="canopy">canopy</li>
-            <span class="point"></span>
-            <li class="backend">Back End Dev</li>
-            <span class="point"></span>
-            <li class="year">2015</li>
-          </ul>
-          <p class="description">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-
-          <ul class="skills">
-            <li class="htl"><a href="#"></a>html</li>
-            <li class="css"><a href="#"></a>css</li>
-            <li class="js"><a href="#"></a>javascript</li>
-          </ul>
-          <button class="seeproject" data-modal-target="#modal1">See Project</button>
-        </div>
-      </div>
-
-      <div class="container2 desktop-reverse" id="container2" data-modal-target="#modal0">
-        <img
-          class="desktop-image"
-          src="Images/Desktop-img2.png"
-          alt="Project Image"
-        />
-        <img class="snapshoot" src="Images/Snapshoot2.png" alt="mobile image" />
-        <div class="daily">
-          <h2 class="Tonic2">Multi-Post Stories</h2>
-          <ul class="offer">
+    <section class="work-container" id="container">
+      <div class="container1" id="portfolio">    
+      <img class="desktop-image" src="Images/Desktop-img1.png" alt="Desktop Image">
+      <img class="snapshoot" src="Images/Snapshoot1.png" alt="mobile image">
```

Comment:
- [ ] Good job so far implementing the mobile menu. :+1: :100: , But I think there are still some changes you need to make before you can move to the next milestone.
- According to the project requirement, when you click on any of the menu items, it is supposed to close the menu and also take you to the specific part of the page, Currently, it only takes you to that specific section but doesn't close the menu, Please look into this. :+1: :wave: 
- Also the mobile menu is meant to cover the full page on a scroll, it should not scroll with the page. :smile: :smile: 
- VIDEO FOR REFERENCE:

https://user-images.githubusercontent.com/107632780/232637662-1ab48de1-fbb8-44c7-a52a-d1d04c154608.mp4